_opcodes/op012-bgee.html

```diff
@@ -66,14 +66,7 @@
 <br/>134217729   Stunning Set to Value
 <br/>134217730   Stunning Set to Percentage
 <br/>134217731   Stunning Reduce by Percentage
-<br/>268435456   Soul Eater
-<br/>268435457   Soul Eater Set to Value
-<br/>268435458   Soul Eater Set to Percentage
-<br/>268435459   Soul Eater Reduce by Percentage
-<br/>1073741824  Disease
-<br/>1073741825  Disease Set to Value
-<br/>1073741826  Disease Set to Percentage
-<br/>1073741827  Disease Reduce by Percentage
```

Comment:
These are iwd damage types — are you sure they removed them in iwdee?

Reply:
Yep, those are exclusive to IWD2.